test/nuts/repros/partialBundleDelete/force-app/digitalExperiences/site/Xcel_Energy1/sfdc_cms__view/error/content.json

```diff
@@ -0,0 +1,71 @@
+{
+  "type": "sfdc_cms__view",
+  "title": "Error",
+  "contentBody": {
+    "sfdc_cms:component": {
+      "definitionName": "community_layout:sldsFlexibleLayout",
+      "sfdc_cms:children": [
+        {
+          "regionName": "content",
+          "sfdc_cms:children": [
+            {
+              "componentAttributes": {
+                "backgroundImageConfig": "",
+                "backgroundImageOverlay": "rgba(0,0,0,0)",
+                "sectionConfig": "{\"UUID\":\"66609d49-c588-40c5-a8d7-755443ed9fb4\",\"columns\":[{\"UUID\":\"b1a85d75-a44b-4392-9ded-5d56b382d087\",\"columnName\":\"Column 1\",\"columnKey\":\"col1\",\"columnWidth\":\"12\",\"seedComponents\":null}]}"
```

Comment:
if only JSON could accept objects and arrays, they wouldn't be forced to serialize this 🤪